library/ssl_tls13_keys.h

```diff
@@ -75,6 +77,9 @@ extern const struct mbedtls_ssl_tls1_3_labels_struct mbedtls_ssl_tls1_3_labels;
     mbedtls_ssl_tls1_3_labels.LABEL,              \
     sizeof(mbedtls_ssl_tls1_3_labels.LABEL)
```

Comment:
Minor: Move this past the new def'n of `MBEDTLS_SSL_TLS1_3_LBL_LEN` and reuse the latter here.